PhysicsTools/PatAlgos/python/slimming/MicroEventContent_cff.py

```diff
@@ -67,6 +67,7 @@
         'keep *_l1extraParticles_*_*',
         'keep L1GlobalTriggerReadoutRecord_gtDigis_*_*',
         # stage 2 L1 trigger
+        'keep *_simGtExtFakeProd__*', 
```

Comment:
By the way, for this PR on 10_6, do we add the additional content directly to MicroEventContent without run2_miniAOD_UL? @rekovic @slava77 
Thanks.

Reply:
we have two control points, in the Task/Sequence definition and in the output.
Historically we did not enforce the EventContent to be in sync to simplify the setup.

Reply:
The above needs change, to keep "by product" not by "module name".  Will commit soon.

Reply:
> Historically we did not enforce the EventContent to be in sync to simplify the setup.

I assume we stay with this.